tox.ini

```
@@ -12,4 +12,5 @@ deps =
   py{36,37,38,39,310}: -rsocial_core/tests/requirements.txt
 commands =
   py{36,37,38,39,310}: pip install -e .[all]
+  py{36,37,38,39,310}: pip install --no-binary lxml lxml
```

I don't think this will work – the above line already installs `lxml` and without `--force-reinstall` nothing will happen here:

```console
$ pip install --no-binary lxml lxml
Requirement already satisfied: lxml in ./.venv/lib/python3.9/site-packages (4.9.2)
```

```suggestion
  py{36,37,38,39,310}: pip install --force-reinstall --no-binary lxml lxml
```